src/components/Accordion/Accordion.tsx

```diff
@@ -40,26 +35,6 @@ export interface AccordionProps extends UIComponentProps<any, any>, ChildrenComp
     title: ShorthandValue
   }[]
 
-  /**
-   * A custom render iterator for rendering each Accordion panel content.
-   * The default component, props, and children are available for each panel content.
-   *
-   * @param {React.ReactType} Component - The computed component for this slot.
-   * @param {object} props - The computed props for this slot.
-   * @param {ReactNode|ReactNodeArray} children - The computed children for this slot.
-   */
-  renderContent?: ShorthandRenderFunction
```

Comment:
In this case, I find this methods useful. They are not callback render for a shorthand, but for each item of array. Do we want to get rid of this as well? It would be nice from user perspective for the array shorthands, to be able to define this logic via an API property. What are your thoughts on this @kuzhelov ?

Reply:
#### Solution with no additional API changes

```jsx
// this could be a helper util
const customTreeShorthand = (value, renderTree) => (render => render(value, renderTree))

const renderCustomMenuItem = (Component, props) => <Component {...props} />

<Menu items={[ 
    { as: ..., ... },
    ...
  ]
 .map(item => customTreeShorthand(value, renderCustomMenuItem) )
} 
.. />
```

#### Solution with `render` method introduced for limited set of components - specifically, collection and hierarchical ones: Menu, Tree, Accordion, List (?)

Needs changes, but we may talk about the final result even now:

```jsx
<Menu items={[ 
    { as: ..., ... },
    ...
  ]}
  renderItem={(Component, props) => ... }  { /* is treated a default rendering logic, if not overwritten by shorthand */ }
.. />
```

What I don't like here is that it is not absolutely clear for the client where `Component` and `props` have come from, and where the data that is provided as shorthand object (i.e. `{ as: ..., ...}` resides in those `Component` and `props`).

Reply:
I am okay with both approaches, but when we have shorthand array, with two shorthands inside, as the panels are inside the Accordion, the user will have to define two maps for both shorthand, which may look awkward... Let's hear other folks' opinion before deciding.